hack/fetch-nvidia-resources.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+#set -o verbose
+
+root=$(dirname "${BASH_SOURCE[0]}")/..
+gpudir=${root}/templates/flavors/nvidia-gpu
+workdir=${gpudir}/tmp
+
+helm repo add nvidia https://nvidia.github.io/gpu-operator \
+    && helm repo update
+helm template gpu-operator nvidia/gpu-operator \
```

Comment:
should we allow version to be passed as a parameter, so that we know what version of operator we are using?